scripts/cleanup-pr.ts

```diff
@@ -187,6 +189,82 @@ class VargasJRAgentCleanup {
     }
   }
 
+  async archiveDevinSession(): Promise<void> {
+    const prDetails = await this.getPRDetails();
+    
+    if (!prDetails) {
+      console.log("⚠️  Skipping Devin session archival - unable to fetch PR details");
+      return;
+    }
+
+    try {
+      const headers = await getGitHubAuthHeaders();
+      const response = await fetch(`https://api.github.com/repos/${prDetails.repository}/pulls/${this.config.prNumber}`, {
+        headers: {
+          ...headers,
+          'User-Agent': 'VargasJR-Cleanup-PR'
+        }
+      });
+
+      if (!response.ok) {
+        console.log(`⚠️  Failed to fetch PR body for session archival: ${response.status}`);
+        return;
+      }
+
+      const prData = await response.json();
+      const prBody = prData.body || '';
+      
+      const sessionUrlRegex = /https:\/\/app\.devin\.ai\/sessions\/([a-f0-9-]+)/i;
+      const match = prBody.match(sessionUrlRegex);
+      
+      if (!match) {
+        console.log("ℹ️  No Devin session URL found in PR description - skipping session archival");
+        return;
+      }
+
+      const sessionId = match[1];
+      console.log(`Found Devin session ID: ${sessionId}`);
+      
+      await this.archiveSession(sessionId);
+      
+    } catch (error) {
+      console.error(`❌ Error during Devin session archival: ${error}`);
+    }
+  }
+
+  private async archiveSession(sessionId: string): Promise<void> {
+    const devinApiToken = process.env.DEVIN_API_TOKEN;
+    
+    if (!devinApiToken) {
+      console.log("⚠️  Skipping Devin session archival - missing DEVIN_API_TOKEN");
+      return;
+    }
+
+    try {
+      console.log(`Archiving Devin session: ${sessionId}`);
+      
+      const response = await fetch(`https://api.devin.ai/v1/sessions/${sessionId}/tags`, {
```

Comment:
This api request should instead be an api where we send a message to to the session that is just 'archive' instead of adding tags